app/views/accounts/index.html.erb

```diff
@@ -0,0 +1,30 @@
+<h1>Accounts</h1>
+ <%= link_to '< Back', groups_path %>
+<section class="section-1">
+  <h2 class="amount">Total Amount: $<%= @accounts_sum %></h2>
+  <% if @accounts.count > 0 %>
+  <div class="table-responsive">
+    <table class="table table-bordered">
+      <thead class="table-light">
+        <tr>
+          <th scope="col">ID</th>
+          <th scope="col">Name</th>
+          <th scope="col">Amount</th>
+        </tr>
+      </thead>
+      <% @accounts.each do |account| %>
+      <tbody>
+        <tr>
+          <th scope="row"><%= account.id %></th>
+          <td><%= account.name%></td>
+          <td>$<%= account.amount%></td>
+        </tr>
+      </tbody>
+      <% end %>
+    </table>
+  </div>
+    <% else %>
+    <h2>No amount</h2>
+  <% end %>
+  <button class="btnLink"><%= link_to "Add Transaction", new_group_account_path %></button>
+</section>
```

Comment:
[REQUIRED] Kindly ensure that the transactions page also matches the template. This also includes the `add transaction` button